README.md

```diff
@@ -36,6 +36,7 @@ auditjs [command]
 Commands:
   auditjs iq [options]    Audit this application using Nexus IQ Server
   auditjs config          Set config for OSS Index or Nexus IQ Server
+  auditjs clear           Clear the cache if a location has been set in config
```

Comment:
Would suggest you do this as `auditjs ossi --clear` because it's a command that's only applicable to OSS Index, in reality. WDYT?

Reply:
Sounds good! I'll start the refactor now

Reply:
in: https://github.com/sonatype-nexus-community/auditjs/pull/153/commits/cb3ff67c7e0f23340e5f58c9dd29a8f2dc8bb826